### ISSUE TITLE
Large Req_Headers Core-dump?

### ISSUE THREAD

**Issue by JoeOrtiz** [fooinha/nginx-json-log#38]
It appears on requests that have large request headers, nginx core dumps, without it, it works fine.

This is all I could see from the nginx debug mode:

```
2018/10/03 20:06:12 [debug] 289#289: *37 post access phase: 11
2018/10/03 20:06:12 [debug] 289#289: *37 generic phase: 12
2018/10/03 20:06:12 [debug] 289#289: *37 generic phase: 13
2018/10/03 20:06:12 [debug] 289#289: *37 http client request body preread 70
2018/10/03 20:06:12 [debug] 289#289: *37 http request body content length filter
2018/10/03 20:06:12 [alert] 286#286: worker process 289 exited on signal 11 (core dumped)
2018/10/03 20:06:12 [debug] 425#425: epoll add event: fd:6 op:1 ev:00002001
2018/10/03 20:06:12 [debug] 425#425: epoll add event: fd:10 op:1 ev:10000001

```

**Comment by fooinha** [comment]
Thanx for this, I'll create a test for this and will investigate.

**Comment by fooinha** [comment]
This may be fixed with #44
